frontend/assets/apis/zafApis.js

```diff
@@ -0,0 +1,39 @@
+import client from "./zafClient.js";
+
+const getCommentTexts = async () => {
```

Comment:
`getCommentsTexts` should call `getComments` instead of reimplementing the logic

Reply:
Fixed! Calling `getComments` would make more sense.